DependencyInjection/MainConfiguration.php

```diff
@@ -51,6 +51,7 @@ public function getConfigTreeBuilder()
         ;
         
         $this->addStoragesSection($rootNode);
+        $this->addStorageEncryptingSection($rootNode);
```

Comment:
that has to go inside `addDynamicGatewaysSection` as we can only encrypt gateway configs for now

https://github.com/Payum/PayumBundle/blob/master/DependencyInjection/MainConfiguration.php#L192